### DISCUSSION TITLE
GPU inference

### DISCUSSION THREAD

**Discussion by BarryKCL** [snakers4/silero-vad#74] [Q&A] [answered]
[W:onnxruntime:Default, fallback_cpu_capability.h:140 GetCpuPreferedNodes] Force fallback to CPU execution for node: Equal_890

**Comment by snakers4** [comment, marked as answer]
These models are not supposed to work on GPU

**Reply by snakers4** [reply]
As for ONNX - I am not even sure we even tried it, I see that now ONNX has it https://pypi.org/project/onnxruntime-gpu/
In any case, these models were designed to use as little compute as possible, hence running them on GPU is not necessary

**Reply by snakers4** [reply]
Also - https://stackoverflow.com/questions/63338386/converted-onnx-model-runs-on-cpu-but-not-on-gpu - can you tell from logs which operations are not supported?

When we working with ONNX error messages were not very informative
